charts/maxscale/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: maxscale
 description: Deploys a maxscale mariadb-galera proxy including mariadb-galera
 type: application
-version: 3.2.8
-appVersion: 6.4.1
+version: 4.0.0
```

Comment:
Is a major version bump really necessary?

Reply:
Ah, nvm, switched this with the `version`.

But you didn't update the image, therefore the `appVersion`shouldn't change either.